Dockerfile

```diff
@@ -9,9 +10,10 @@ RUN apt-get update && env DEBIAN_FRONTEND=noninteractive apt-get install -y \
 RUN git clone https://github.com/jnovy/pxz.git /root/pxz
 RUN cd /root/pxz && make
```

Comment:
Note to self: this is not correctly cross-compiled on CI, the final binary is x86. I will _try_ to fix but considering that this is pure value-add if I don't find an easy solution I'll switch this job to also emulate instead of cross-compile (slower, but only on CI)

Reply:
Fixed in the last force-push, can @mzedel or @janaperic try the following?
```
docker run --rm --entrypoint /usr/bin/pxz registry.gitlab.com/northern.tech/mender/mender-convert:1432652685 fake-file
```

:pray:

Reply:
results in:
```
/usr/bin/pxz: can't stat 'fake-file': No such file or directory
```
is this good?

Reply:
That is good! :partying_face: